netbox/dcim/tests/test_models.py

```diff
@@ -588,3 +588,30 @@ def test_cable_cannot_terminate_to_a_wireless_interface(self):
         cable = Cable(a_terminations=[self.interface2], b_terminations=[wireless_interface])
         with self.assertRaises(ValidationError):
             cable.clean()
+
+
+class CableTestCase(TestCase):
```

Comment:
I assume this was added by mistake? (CableTestCase is declared above.)

Reply:
Eh, I think I half started on the tests and forgot to finish them.  Should be updated soon.